app/views/deposits/fixed/newapplication.html

```diff
@@ -299,9 +306,9 @@
                     </tr>
                 </table>
                 </div>
-                <div class="col-md-offset-5 ">
-                    <button id="cancel" ng-click="cancel()" class="btn btn-default">{{ 'label.button.cancel' | translate }}</button>
-                    <button id="save" type="submit" class="btn btn-primary" ng-show="data">{{ 'label.button.save' | translate }}
+                <div class="col-md-offset-5" ng-show="formData.productId">
```

Comment:
here the cancel and submit (at the bottom) button shows up and the button on line 26-27 hide when a product is selected from product dropdown.